### DISCUSSION TITLE
挑战内容支持多语言展示

### DISCUSSION THREAD

**Discussion by deanplus** [decert-me/website#280] [General]
1. 目前基本上是中文编写的挑战内容。如何支持多语言展示？（不止页面，还涉及到领取NFT时元数据里面的挑战内容）
2. 如何让其他人共同参与翻译？

**Comment by ccxdcyl** [comment]
挑战：
多语言指向同一个NFT
页面上支持翻译显示成其它语言
切换语言时，切换成对应的翻译
元数据的问题（领取时写入对应语言的挑战内容ipfs）

翻译：
翻译IPFS中挑战内容
在github新建issue，待认领
认领（邀请贡献者领取）
合并
处理成新IPFS

**Reply by Oaciqihz** [reply]
github新建issue是怎么新建呢？手动新建吗？

**Reply by deanplus** [reply]
![截屏2023-12-11 10 35 37](https://github.com/decert-me/website/assets/121151857/f8413d11-3dff-4c63-b391-b1ae05f0d4eb)
这里应该有一个操作文档，方便别人参与。

理论上大家都可以创建issue

**Comment by deanplus** [comment]
@Oaciqihz 制定一下整体流程

**Comment by Oaciqihz** [comment]
@0xdwong 
![image](https://github.com/decert-me/website/assets/108737430/2cfcf0ed-1f86-46aa-b4d9-73f8f8464b5b)

**Reply by deanplus** [reply]
got you

**Comment by Oaciqihz** [comment]
参与翻译流程:
https://github.com/decert-me/document/wiki/%E5%8F%82%E4%B8%8E%E7%BF%BB%E8%AF%91
